spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/src/main/java/org/springframework/cloud/dataflow/common/test/docker/compose/execution/Executable.java

```diff
@@ -21,6 +21,6 @@ public interface Executable {
 
     String commandName();
 
-    Process execute(String... commands) throws IOException;
+    Process execute(boolean composeCommand, String... commands) throws IOException;
```

Comment:
We probably need to have javadocs here explaining what the composeCommand means.